src/test/java/fr/paris/lutece/portal/service/daemon/DaemonSchedulerTest.java

```diff
@@ -360,13 +368,17 @@ private void testScheduleTwice( boolean shouldThrow ) throws ClassNotFoundExcept
             testDaemon.waitForCompletion( );
             assertTrue( testDaemon.hasRun( ) );
         }
+        catch( InterruptedException | BrokenBarrierException | TimeoutException e)
+        {
+            fail("failed" + e + ":" + e.getMessage( ) );
```

Comment:
here you swallow the exception and you loose the stack trace which might be helpful. Why not let tests throw exceptions, which makes them fail ?